### DISCUSSION TITLE
We need documentation!

### DISCUSSION THREAD

**Discussion by ronsilverentand** [appcompany/versioning-api#115] [Ideas]
We used these commands today to get the GitHub package registry to work, this NEEDS to be documented.

npm login --scope=@onezerocompany --registry=https://npm.pkg.github.com

npm config set @onezerocompany:registry https://npm.pkg.github.com

chmod +x .husky/commit-msg

**Comment by lucasilverentand** [comment]
I think this should all be moved to a general repository that contains all kinds of coding guides.
